src/Toolbar.tsx

```diff
@@ -329,9 +329,6 @@ const ToolbarItemContents = memo(function ToolbarItemContents({
                     target="_blank"
                     rel="noreferrer"
                     className="flex items-center rounded-sm p-1 no-underline text-inherit hover:bg-primary/10 hover:text-primary dark:hover:bg-chalkboard-70 dark:hover:text-inherit"
-                    onClickCapture={(e) =>
-                      e.nativeEvent.stopImmediatePropagation()
-                    }
```

Comment:
I couldn't see anything obvious reason why this needed to be there, I didn't notice anything weird happening with it gone.

Reply:
Oh that's because the selection and other events on the stream or any element below the tooltip will fire if you click the link normally